src/components/reusable/notification/notification.scss

```diff
@@ -47,13 +70,53 @@ kd-card.notification-mark-read::part(card-wrapper) {
     align-items: center;
     padding-top: 10px;
   }
+  &-state-icon {
+    padding-right: 8px;
+  }
+}
+
+.notification-inline .notification-head,
+.notification-toast .notification-head {
+  display: flex;
+  align-items: center;
 }
 
+.notification-inline .notification-content-wrapper,
+.notification-toast .notification-content-wrapper {
+  padding: 0;
+}
 .notification-description ::slotted(*) {
   @include typography.type-ui-02;
   color: var(--kd-color-text-primary);
   padding-top: 10px;
 }
+.notification-inline .notification-description ::slotted(*),
+.notification-toast .notification-description ::slotted(*) {
```

Comment:
I don't really like the use of `::slotted(*)` here, it is too broad and applies to any slotted element. Can we remove all of these * rules?

Reply:
Then how to style this slotted part for description. As we added inline and toast inside <kyn-notification> so I remain UI as it is and handled things by styles. Will try to reduce if possible but can't remove all of these

Reply:
It depends what you're trying to accomplish, but you shouldn't have to style any slotted elements. Styling an internal element that wraps the slot should be sufficient.

Reply:
ok will change that logic of scss.

Reply:
Done Removed `::slotted(*)`